Added a link to the IRC server

soarn opened the pull request:
<a href="https://www.irccloud.com/invite?channel=%23lobby&amp;hostname=irc.cahbot.pro&amp;port=6697&amp;ssl=1" target="_blank">Join the #lobby channel on irc.cahbot.pro!</a>
kinda suprised markdown actually accepted the code

soarn committed:
Added a link to the IRC server

<a href="https://www.irccloud.com/invite?channel=%23lobby&amp;hostname=irc.cahbot.pro&amp;port=6697&amp;ssl=1" target="_blank">Join the #lobby channel on irc.cahbot.pro!</a>
kinda suprised markdow actually accepted the code

2003cah commented:
Since CahBot is not related to IRC (IRCahBot is, of course), I don't see the need for this

soarn committed:
Changed IRC channel

<a href="https://www.irccloud.com/invite?channel=%23ircahbot&amp;hostname=irc.cahbot.pro&amp;port=6697&amp;ssl=1" target="_blank">Join the #ircahbot channel on irc.cahbot.pro</a>

Chewsterchew commented:
You do make a good point cah

2003cah commented:
Me? Making good points? What?

Chewsterchew commented:
can i be mod

advaith1 commented:
same

advaith1 commented:
I was an admin once
